Jenkinsfile

```diff
@@ -7,8 +7,7 @@ buildMvn {
   doDocker = {
     buildJavaDocker {
       publishMaster = true
-      healthChk = true
-      healthChkCmd = 'wget --no-verbose --tries=1 --spider http://localhost:8081/admin/health || exit 1'
+      // healthChk is in FolioDcbApplicationIntegrationTest.java
```

Comment:
Let's be careful about disabling the health check here. Need to understand what the consequences will be for the community builds.

Reply:
Checked with @funkymalc  and removing the healthcheck from the Jenkinsfile is fine.